### ISSUE TITLE
Error when using dev formula for v1.1.113

### ISSUE THREAD

**Issue by pat-s** [rundel/homebrew-quarto-cli#37]
When invoking quarto from the dev formula, I get the error below (v1.1.113). 
This does not occur using the release formula (v1.0.38) or when using the normal `.pkg` installer.

```bash
quarto preview
usage: basename string [suffix]
       basename [-a] [-s suffix] string [...]
ERROR: Cannot determine the Quarto source path. This script must be run from the bin or common folder.
```

**Comment by rundel** [comment]
Looks like something changed with the recent dev versions of quarto - these formulae are installing into /opt/homebrew/ which is not typical for quarto and seems to conflict for the moment. Let me take a look at the recent changes and see if I can track down a cause.

**Comment by rundel** [comment]
See above for the issue opened on quarto-dev/quarto-cli - it looks like quarto-dev/quarto-cli#2003 introduced the breaking change. I'm not quite sure how to fix things (without breaking some other edge cases) but it seems like something that should be fixable. At least for now the non-dev version is still working.

**Comment by rundel** [comment]
This should be fix with v1.1.120 which is now available

**Comment by pat-s** [comment]
Work again, thanks for the quick fix!
